### DISCUSSION TITLE
No ejecuta el .jar haciendo doble clic directo

### DISCUSSION THREAD

**Discussion by francorojo** [dds-jv/foro#145] [Consultas de herramientas] [answered]
Buenaas tenia una duda, la verdad q buscamos mucho pero no logramos que arranque directo el archivo .jar con el doble clic.
Desde consola arranca sin problemas de la siguiente forma: java -jar path
Pero desde el archivo directo .jar, no agarra, probando abriendo con distintas aplicaciones desde la carpeta de java, como java.exe, javaw.exe y javaws.exe y otras.
Hice un archivo .cmd para que directamente se ejecute el comando java -jar "...\archivo.jar" y ahi tambien arranca.
Actualice java pero tampoco. Sera algun problema nuestro que nos falto agregar, o tal vez tengo q cambiar algo de java o la verdad no se, sabran algun motivo por el cual no ejecuta solo desde el doble clic?
(No sabria que informacion adicional subir del pom o algo, si se necesita algo que mostrar me avisan y subo)
Muchas Graciaaas!

**Comment by gastonprieto** [comment]
Buenas!!

No voy a mentirte Marge.... pero jamás lo hice :innocent: , aunque acá tiran un par de comentarios que tal vez sirvan: https://www.pixelstech.net/article/1463037958-Run-executable-jar-on-double-click-on-Windows
La única vez que hice algo parecido hice un `.bat` que ejecutaba el comando `java -jar archivo` como decias vos.

La buena noticia es que nos alcanza que lo ejecute por linea de comandos. :sunglasses:

**Comment by francorojo** [comment, marked as answer]
Buenisimo jajajajaj, muchas gracias!!!
